### DISCUSSION TITLE
i cannot install chocotaley on my desktop and laptop

### DISCUSSION THREAD

**Discussion by Baidbomba10** [chocolatey/choco#2497] [General] [answered]
STOP RIGHT HERE - DO NOT CREATE A TICKET FOR Chocolatey GUI ON THIS REPOSITORY

This is the repository for choco.exe, the command line interface for Chocolatey, and issues/feature suggestions should be limited to only choco.

For Chocolatey GUI, you will need to head over to https://github.com/chocolatey/ChocolateyGUI and file a ticket there.

NOTE: Keep in mind we have an etiquette regarding communication that we expect folks to observe when they are looking for support in the Chocolatey community. https://github.com/chocolatey/choco/blob/develop/README.md#etiquette-regarding-communication

**Comment by Baidbomba10** [comment]
please help me install chocolatey

**Comment by TheCakeIsNaOH** [comment, marked as answer]
@Baidbomba10 Here are the installation instructions:
https://chocolatey.org/install

If you have further issues I'd suggest joining the chat:
https://ch0.co/community
